corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -574,43 +575,32 @@ def get_username_in_last_form_user_id_submitted(domain, user_id):
         return user_submissions[0]['form']['meta'].get('username', None)
 
 
-def get_wrapped_ledger_values(domain, case_ids, section_id, entry_ids=None, pagination=None):
-    # todo: figure out why this causes circular import
-    from corehq.apps.reports.commtrack.util import StockLedgerValueWrapper
-    query = (LedgerES()
-             .domain(domain)
-             .section(section_id)
-             .case(case_ids))
-    if pagination:
-        query = query.size(pagination.count).start(pagination.start)
-    if entry_ids:
-        query = query.entry(entry_ids)
-
-    return [StockLedgerValueWrapper.wrap(row) for row in query.run().hits]
+def get_wrapped_ledger_values(domain, case_ids, section_id, entry_ids=None):
```

Comment:
the test that is failing on this, is failing because its test is not actually saving ledgers, but instead sending raw json to the ledger ES index. these functions should get moved out of esaccessors anyways